### PR TITLE
Rule: SC4-1-2-aria-hidden-focus

### PR DESCRIPTION
**Pull request by WilcoFiers**
# Checklist for Pull Request
🚨Please review the [guidelines for contributing](https://auto-wcag.github.io/auto-wcag/pages/contribute.html) to this repository.

- [ ] Make sure you are requesting to **pull a issue/feature/bugfix branch** (right side). Don't request your master branch!
- [ ] Make sure you are making a pull request against the **master branch** (left side). Also you should start *your branch* off *master branch*.
- [ ] Kindly prefix the pull request name with `DRAFT` or `FINAL` accordingly. For more details, read [Rule Design Guidelines](https://auto-wcag.github.io/auto-wcag/pages/design/review.html)
- [ ] Provide a reference to the issue being addressed.

#Reviewing Guidelines
- For `DRAFT` PR's, leave any comments you have and approve if you’ve reviewed the entire pull request. 
- Once there have been three such reviews the pull request can be merged. 
- Any comments left will be taken up in an updated pull request, which will be necessary before a FINAL release”.
- For `FINAL` PR's, review and approve changes to be merged.

# How to Review And Approve
- Go to the “files changed” tab, there you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “approve” and click “Submit review”

### Description
Please describe your pull request.

💔Thank you!

### CODE REVIEW

File: _rules/SC4-1-2-aria-hidden-focus.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+By adding `aria-hidden="true"` to an element, content authors ensure that assistive technologies will ignore the element. This can be used to hide decorative parts of a web page, such as icon fonts - that are not meant to be read by assistive technologies.
+
+A focusable element with `aria-hidden="true"` is ignored as part of the reading order, but still part of the focus order, making it's state of visible or hidden unclear.
+
+- https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
```
Comment (jkodu) [inline _rules/SC4-1-2-aria-hidden-focus.md]: These needs to be links.

File: _rules/SC4-1-2-aria-hidden-focus.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+authors:
+- Wilco Fiers
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any element with a `aria-hidden="true"` attribute.
```
Comment (jkodu) [inline _rules/SC4-1-2-aria-hidden-focus.md]: editorial: with an `aria-hidden=true`....

File: _rules/SC4-1-2-aria-hidden-focus.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+**Note**: Using `aria-hidden="false"` on a decendent of an element with `aria-hidden="true"` **does not** expose that element. `aria-hidden="true"` hides everything inside of it from assistive technologies.
```
Comment (jkodu) [inline _rules/SC4-1-2-aria-hidden-focus.md]: editorial: hides itself and its children element from ..... rather than everything inside?

**Comment by WilcoFiers** [comment]
@ShadowBB @kasperisager could you review this PR?

**Pull request merged**